app/src/main/java/com/example/techstacksetup/List.java

```diff
@@ -71,10 +72,13 @@ public void onSuccess(QuerySnapshot queryDocumentSnapshots) {
                             String name = user.getName();
                             //Data is a large string with a new line after every entry
                             data += "Name: " + name + "\n";
-                            nameArray.add(data);
+                            nameArray.add(name);
```

Comment:
String data is no longer in use and can be removed

Reply:
I left all that there just incase we need to reference back to it in cap project